src/GovUk.Education.ExploreEducationStatistics.Data.Services/ViewModels/Meta/ObservationalUnitMetaViewModel.cs

```diff
@@ -11,7 +11,7 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Services.ViewModels.Me
     /// </summary>
     public record ObservationalUnitMetaViewModel : LabelValue
     {
-        [JsonConverter(typeof(StringEnumConverter), true)]
+        [JsonConverter(typeof(StringEnumConverter), typeof(CamelCaseNamingStrategy))]
```

Comment:
What's the camelcase strategy for?

Reply:
Yes I think it's to meet the required case of the `GeographicLevel` field expected by the front end.

The reason for this change though is as follows:

The second parameter of the `JsonConverterAttribute` is the parameter list to use when constructing the converter, in this case a `StringEnumConverter`.

The constructor overload for `bool` is obsolete. So instead we are just updating it use a different constructor.
![image](https://user-images.githubusercontent.com/4147126/147175366-1ea5777d-0991-4f78-8464-ad2f6c77350c.png)

The result should be exactly the same behaviour. I made this change in one other place as well.

Reply:
Nice one, thanks!